### PR TITLE
docs: Updated README with information about the authentticators

### PR DESCRIPTION
**Pull request by padamstx**
BREAKING CHANGE: this release introduces a new authentication scheme

### CODE REVIEW

**Comment by padamstx** [comment]
tagged @dpopp07 and @mediumTaj mainly for awareness about the readme/authentication updates

File: Authentication.md [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+// Create the service options struct.
+options := &exampleservicev1.ExampleServiceV1Options{
+    Authenticator: authenticator,
+}
+
+// Construct the service instance.
+service := exampleservicev1.NewExampleServiceV1(options)
+
+// 'service' can now be used to invoke operations.
+```
```
Comment (dpopp07) [inline Authentication.md]: I wonder if this shouldn't be `AUTHENTICATION.md` - I feel like most markdown files use all caps. Really not a big deal though, just thinking out loud

File: .bumpversion.cfg [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.8.0
```
Comment (ehdsouza) [inline .bumpversion.cfg]: Take the latest version from master

File: Authentication.md [hunk 3]
```diff
@@ -0,0 +1,349 @@
+# Authentication
+The go-sdk-core project supports the following types of authentication:
+- Basic Authentication
+- Bearer Token 
+- Identity and Access Management (IAM)
+- Cloud Pack for Data
```
Comment (ehdsouza) [inline Authentication.md]: I think this is `Cloud Pak for Data`

File: core/version.go [hunk 4]
```diff
@@ -15,4 +15,4 @@ package core
 // limitations under the License.
 
 // Version of the SDK
-const __VERSION__ = "0.8.0"
```
Comment (ehdsouza) [inline core/version.go]: Lets stick to the master version

**Comment by padamstx** [comment]
Pushed an amended commit with the requested updates

**Pull request merged**